server/base/src/main/java/org/apache/accumulo/server/constraints/MetadataConstraints.java

```diff
@@ -201,7 +203,9 @@ public List<Short> check(Environment env, Mutation mutation) {
       }
 
       if (columnUpdate.getValue().length == 0 && !columnFamily.equals(ScanFileColumnFamily.NAME)
-          && !columnFamily.equals(OnDemandAssignmentStateColumnFamily.NAME)) {
+          && !(columnFamily.equals(HostingColumnFamily.REQUESTED_COLUMN.getColumnFamily())
+              && columnQualifier
+                  .equals(HostingColumnFamily.REQUESTED_COLUMN.getColumnQualifier()))) {
```

Comment:
This can be shortened.

```suggestion
          && !HostingColumnFamily.REQUESTED_COLUMN.equals(columnFamily,columnQualifier)) {
```

Reply:
Suggestion applied in cd223b6